chain/cosmos/chain_node.go

```diff
@@ -852,9 +852,17 @@ func (tn *ChainNode) SendIBCTransfer(
 	if options.Timeout != nil {
 		if options.Timeout.NanoSeconds > 0 {
 			command = append(command, "--packet-timeout-timestamp", fmt.Sprint(options.Timeout.NanoSeconds))
-		} else if options.Timeout.Height > 0 {
+		}
+
+		if options.Timeout.Height > 0 {
 			command = append(command, "--packet-timeout-height", fmt.Sprintf("0-%d", options.Timeout.Height))
 		}
+
+		if options.AbsoluteTimeouts {
```

Comment:
I also considered removing this from the options type and adding the `--absolute-timeouts` flag if a timeout height was set, but this seemed more explicit

Reply:
I like this more so you get the type safety as well